### ISSUE TITLE
(an attendee Data Disco workshop 28 Nov) Suggestion we explicitly default the metadata to CC-0 (or equivalent)

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#26]
i.e. set ISO 19115 MD_Metadata.metadataConstraints to https://creativecommons.org/publicdomain/zero/1.0/ or ….
       <gmd:MD_LegalConstraints>
          <gmd:useConstraints>
            <gmd:MD_RestrictionCode 
              codeList='http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/gmxCodelists.xml#MD_RestrictionCode' 
              codeListValue='otherRestrictions'>otherRestrictions</gmd:MD_RestrictionCode>
          </gmd:useConstraints>
          <gmd:otherConstraints>
      <gmx:Anchor xlink:href='http://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse/noConditionsApply'>no conditions apply</gmx:Anchor>
          </gmd:otherConstraints>
        </gmd:MD_LegalConstraints>

**Comment by nmtoken** [comment]
Should we use full name in title not disco

**Comment by 6footdestiny** [comment]
Which 'parts' of GEMINI - the docs were already releases under some permissive license I think? CC-0 for the 'code' feels less sensible to me.

**Comment by PeterParslow** [comment]
This isn't about parts of GEMINI - which as you say are released under CC-BY.

This is about encouraging GEMINI users to explicitly state the licence under which their metadata records can be used, with the default/presumption being that metadata is 'free to be reused'.

**Comment by 6footdestiny** [comment]
In which case and given the primary constituency I would have thought OGL was more appropriate?

**Comment by PeterParslow** [comment]
Perhaps - but not every GEMINI metadata creator is "government". Also, OGL requires attribution, so anyone re-using the metadata would be required to state where they got it from. That may or may not be a good thing. What do others think - about the idea, and about which licence to suggest?

**Comment by nmtoken** [comment]
I'm not sure CC-0 would be a correct default 

> You can copy, modify, distribute and perform the work, even for commercial purposes, all without asking permission. 

(https://creativecommons.org/publicdomain/zero/1.0/) 

Because generally we don't want people to modify the metadata.

**Comment by PeterParslow** [comment]
That is a good point: 

Here's an example that has come across my in-tray today: a certain OS customer ingests OS data into their system, and in parallel to that creates themselves a metadata record that is based on our GEMINI record. It is import that this is a new metadata record - derived from ours - not a modification of the existing one. Specifically, we don't want them keeping the same _fileIdentifier_ if they change anything else (e.g. adding an extra "point of contact" to someone internal to their organisation)!

**Comment by PeterParslow** [comment]
I'm not sure that CC has a variant which allows "remixing" (as opposed to modification of the original) without requiring "attribution"!

**Comment by nmtoken** [comment]
Is that an issue?  

The metadata has to include the metadata point of contact, so will always be attributed.

**Comment by PeterParslow** [comment]
> The metadata has to include the metadata point of contact, so will always be attributed.

The attribution requirement in any of the Creative Commons license is that the derived work attributes the original author. I would expect the _Metadata point of contact_ of the derived work to be the relevant contact for that derived work, not for the original. 

I don't think that CC-0 gives someone the right to modify the original; the problem is that we (the GEMINI record creator) would want to specify that someone creating a derived work HAS TO give it a new _File identifier_. We may actually be less concerned about being attributed - and although the Creative Commons site says that all their licences require attribution, I can't see that CC-0 does (because they see it as 'not a licence').

I think it's a custom licence (CC-0 or CC-BY plus 'shall use a new _File identifier_); perhaps we'd have to roll our own, adding that one obligation?

**Comment by nmtoken** [comment]
What if the default was CC-BY-ND, because generally we don't want to allow derived (meta)data

> You are free to:
Share — copy and redistribute the material in any medium or format
for any purpose, even commercially

But metadata providers could  also enter into other arrangements (“dual licensing”) to allow derivations and remixing, such as different file identifiers 

(https://creativecommons.org/faq/#can-i-enter-into-separate-or-supplemental-agreements-with-users-of-my-work)

**Comment by PeterParslow** [comment]
That may be better, but:
- we (or at least, each publisher) would have to state how to satisfy the CC-BY-ND obligation to 'give appropriate credit'. I guess we can simply state the fact that because the original GEMINI record already contains the _Metadata point of contact_, that is sufficient to satisfy this obligation
- it does allow 'remix, transform, or build upon', just that the result can't be re-distributed. I guess that's OK: but the organisation that does that (without changing the _File identifier_) could get in a mess!

**Comment by archaeogeek** [comment]
Taking this back to first principles since I wasn't at the meeting where this was raised- what's the context for this? What are we (as data and metadata producers) trying to protect (or protect against)? I think we have to be very careful about how this is worded to avoid confusion for end-users (we can't assume any familiarity whatsoever with CC licensing)

**Comment by PeterParslow** [comment]
As someone who was at the meeting, I think it is not so much to protect/protect against, as to encourage / enable / facilitate.

Apparently, there are some people who are unsure that they can take a metadata record, ingest it into their system, use it as a base for any sort of processing - including creating derived metadata records to describe derived datasets. The guy who raised this concern suggested that explicitly stating the licence under which the metadata record is made available could ease this situation.

I felt (& I think other "GEMINI folk" present at the time) felt that 'of course that's what you should do', hence the idea that GEMINI records default to allowing that kind of thing.

**Comment by nmtoken** [comment]
Just revisiting this and moving forward BGS metadata will be explicit in stating metadata is provided under CC BY-ND 4.0.

**Comment by archaeogeek** [comment]
This might be better as guidance rather than an explicit instruction. Assigning to @nmtoken with assistance from @PeterParslow to create initial page

**Comment by nmtoken** [comment]
Intending to implement in XML like:

    <gmd:resourceConstraints xlink:title="MetadataConditions">
        <gmd:MD_LegalConstraints>
            <gmd:useConstraints>
                <gmd:MD_RestrictionCode codeList="gmxCodelists.xml#MD_RestrictionCode"
                    codeListValue="otherRestrictions"/>
            </gmd:useConstraints>
            <gmd:otherConstraints>
                <gmx:Anchor xlink:href="https://creativecommons.org/licenses/by-nd/4.0/">Metadata is
                    distributed with a CC BY-ND 4.0 (Attribution-NoDerivs 4.0 International)
                    licence</gmx:Anchor>
            </gmd:otherConstraints>
        </gmd:MD_LegalConstraints>
    </gmd:resourceConstraints>

**Comment by nmtoken** [comment]
You are right! that would be better so...

    <gmd:metadataConstraints>
      <gmd:MD_LegalConstraints>
         <gmd:useConstraints>
            <gmd:MD_RestrictionCode codeList="gmxCodelists.xml#MD_RestrictionCode"
               codeListValue="otherRestrictions"/>
         </gmd:useConstraints>
         <gmd:otherConstraints>
            <gmx:Anchor xlink:href="https://creativecommons.org/licenses/by-nd/4.0/">Metadata is
               distributed with a CC BY-ND 4.0 (Attribution-NoDerivs 4.0 International)
               licence</gmx:Anchor>
         </gmd:otherConstraints>
      </gmd:MD_LegalConstraints>
    </gmd:metadataConstraints>

**Comment by PeterParslow** [comment]
Now we need to agree where to put that XML fragment and how to introduce what it's about!

1. Where to put it?
I think it belongs in 2.2.14 https://agiorguk.github.io/gemini/1048-uk-gemini-encoding-guidance.html#_limitations_conditions_and_licences, alongside the other "licence" bits. It's almost another sub-section in / near there.

It would need some text, such as:

"ISO 19115 allows a specific license declaration for the metadata itself. Generally, part of the reason to publish structured metadata is so that others can ingest it and manage it within their systems alongside the data, you may want to use something like CC BY-ND, which allows people to transform / remix the metadata but not to re-distribute the result. You should encourage anyone who does transform the metadata to create and use a new File identifier. 
The ISO 19115 metadata constraint element should appear towards the end of the file, after all the GEMINI elements. This position is set by the ISO 19139 schemas, based on the order implied in the ISO 19115 standard."

This approach deliberately avoids adding "Metadata constraints" as an additional GEMINI element. But it does break new ground by giving advice concerning a bit of ISO 19115 which isn't actually in GEMINI - so it may be "cleaner" to add a new element, id = 54.

2. we could introduce the whole matter of 'enabling & controlling what people do with your metadata record" in https://agiorguk.github.io/gemini/1052-metadata-guidelines-for-geospatial-data-resources-part-1.html. Probably as part of a section on "the purpose(s) of metadata"!

**Comment by archaeogeek** [comment]
Agreed approach is fine, but from an implementer's perspective (eg GeoNetwork) some changes would need to be made to allow the element to be easily used in the editor

**Comment by archaeogeek** [comment]
Agreed to add new element. @archaeogeek to start list of sub-issues for related tasks

**Comment by archaeogeek** [comment]
@alexarobu  could you take a look at this, from the perspective of what would be needed in GeoNetwork to implement the additional element, as per https://github.com/agiorguk/gemini/issues/26#issuecomment-2092630145 and following comments? We don't need anything formal, just a comment with the steps needed.

**Comment by alexarobu** [comment]
@archaeogeek I think the following needs to happen, but sanity-check this, as it's quite uncharted territory for me 🙈

- an element for gmd:metadataConstraints needs to be defined in src/main/plugin/iso19139.gemini23/schema/gmd
- a validation rule needs to be added to src/main/plugin/iso19139.gemini23/schematron, I suppose this would go into the supplemental rules, not the main set
- a field needs to be defined in src/main/plugin/iso19139.gemini23/layout/config-editor.xml for the editing view
- the field needs to be added to the record view in src/main/plugin/iso19139.gemini23/formatter (full view as well as default view)
- sample records need to be updated or a new sample added in src/main/plugin/iso19139.gemini23/sample-data
- same for the templates in src/main/plugin/iso19139.gemini23/templates
- the element needs to be included in the output to CSW in src/main/plugin/iso19139.gemini23/present - we need to be aware of any implications this will have for harvesting to DGu, MEDIN, etc as it may cause errors
- at least src/main/plugin/iso19139.gemini23/loc/eng needss to be updated for the new element, ideally the other languages too
- I don't really understand what's happening in src/main/plugin/iso19139.gemini23/index-fields but I assume ElasticSearch needs to be aware of the new field
- the suggest.xsl may also need adjusting to include the new element and the inflate-metadata.xsl in src/main/plugin/iso19139.gemini23

**Comment by archaeogeek** [comment]
I think https://github.com/agiorguk/gemini/issues/26#issuecomment-4378784929 is correct, which is a lot of work!

Am I right in thinking this would be a breaking change? TBH regardless I think it belongs in a new release of the standard.

**Comment by alexarobu** [comment]
@archaeogeek I agree, it feels like a breaking change. I think considering it as part of a future version of GEMINI is probably wiser

**Comment by nmtoken** [comment]
I don't see it as a breaking change to the standard; a metadata record without this content will still be valid.  It would be a minor release, an addition to the standard.

**Comment by archaeogeek** [comment]
Confirmed- to be pushed to the next minor release
